lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize (name_arg, capacity_arg)
+    @name = name_arg
+    @capacity = capacity_arg
+    @patrons = []
+  end
+
+  def add_patron (name)
+    patrons.push(name)
+  end
+
+  def yell_at_patrons
+    capitilized_names = []
+    patrons.each do |patron_name|
+      capitilized_names.push(patron_name.upcase)
+    end
+    return capitilized_names
```

Comment:
You don't need the keyword `return` here. You can just write `capitilized_names` as the last line